Sources/StreamVideo/OpenApi/generated/Models/RingSettings.swift

```diff
@@ -1,33 +1,36 @@
 //
-// RingSettings.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct RingSettings: Codable, JSONEncodable, Hashable {
+    
+public final class RingSettings: @unchecked Sendable, Codable, JSONEncodable, Hashable {
+    
     public var autoCancelTimeoutMs: Int
     public var incomingCallTimeoutMs: Int
+    public var missedCallTimeoutMs: Int
```

Comment:
A bit dangerous as well, since it's newly added - are we guaranteed that it's always there?